doc/platform-support.appendix.md

```diff
@@ -0,0 +1,11 @@
+# Platform Support {#appendix-platform-support}
+
+Currently, the `x86_64-linux` platform receives the highest level of support, both in terms of CI and of developer attention.
+The `aarch64-linux` plaform is also well supported, with most packages building succesfully in CI.
+
+On Darwin, `aarch64-darwin` receives better support than `x86_64-darwin`.
```

Comment:
Isn't `x86_64-darwin` and `aarch64-darwin` situation more complicated than really comparable levels because from time to time CI before merging actually works better for the former?

Reply:
@toonn may have a thought on this.

Reply:
I can't really disagree with the statement. Most of NixOS/darwin-maintainers seem to be aarch64-darwin users. The Darwin builders for Hydra are aarch64-darwin for the most part, emulating x86_64-darwin when necessary. Maybe a qualifier like "slightly" is appropriate. The goal is for them to be equal for the foreseeable future.